src/usecases/verifyWardCode.js

```diff
@@ -1,20 +1,14 @@
 import logger from "../../logger";
 
-const verifyWardCode = ({ getDb }) => async (wardCode) => {
-  const db = await getDb();
-
+const verifyWardCode = ({ getFindWardByCodeGateway }) => async (wardCode) => {
```

Comment:
More of a question (also slacked you this but might be useful to share answer if others were wondering). At the moment the gateway will always be the postgres one with those SQL queries - in the future when we have both a pg and mssql gateway for the same query how will the correct gateway be decided?

Reply:
Either the app container will choose which gateway to use or we will replace these gateways entirely